config/locales/idv/en.yml

```
@@ -233,6 +233,16 @@ en:
         resend_timeframe: Letters typically take 3 to 7 business days to arrive.
         timeframe_html: Letters are sent the next business day via USPS First Class Mail
           and <strong>typically take 3 to 7 business days to arrive</strong>.
+      gpo_reminder:
+        body_html: You requested a letter to verify your identity on
+          <strong>%{date_letter_was_sent}</strong>. You’ll need to enter the
+          code from the letter to finish verifying your identity. Sign back in
+          to %{app_name} to finish verifying your identity.
+        did_not_get_a_letter_html: If you didn’t get this letter, <span
+          class=shut-up-lint></span>%{another_letter_link}
```

I think the solution here is to name rename the interpolation instead of adding this span:
```suggestion
        did_not_get_a_letter_html: If you didn’t get this letter, %{another_letter_link_html}
```

---

Changed, and thank you.